src/ci/docker/host-x86_64/dist-riscv64-linux/crosstool-ng.sh

```diff
@@ -6,7 +7,7 @@ curl -Lf $url | tar xzf -
 cd crosstool-ng-crosstool-ng-1.24.0
 ./bootstrap
 ./configure --prefix=/usr/local
-make -j$(nproc)
+make -j"$(nproc)"
```

Comment:
There's no point in this; if `nproc` prints something that's not a number make will error anyway.